zebra-chain/benches/redpallas.rs

```diff
@@ -5,8 +5,11 @@
 
 use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion, Throughput};
 use rand::{thread_rng, Rng};
-
-use zebra_chain::primitives::redpallas::*;
+use reddsa::{
```

Comment:
We should move this benchmark out of `zebra-chain` into `reddsa` now, the existing bench in `reddsa` is just the Sapling (RedJubjub) variant

Reply:
(This is not a blocker to this PR merging but it's a thing we should track)

Reply:
Can you open a Zebra or `reddsa` ticket?
Whatever is easier for the `reddsa` developers and Pili to track.

Reply:
Created https://github.com/ZcashFoundation/reddsa/issues/42